CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ releases, in reverse chronological order.
 v2.2.0
```

Comment:
Since we're dropping `--urgent`, we're breaking backward compatibility, so we'd have to push this to `3.0.0`. Please update this as part of the PR.